app/models/vim_performance_state.rb

```diff
@@ -3,7 +3,8 @@ class VimPerformanceState < ApplicationRecord
 
   belongs_to :resource, :polymorphic => true
 
-  ASSOCIATIONS = [:vms, :hosts, :ems_clusters, :ext_management_systems, :storages, :container_nodes, :container_groups]
+  ASSOCIATIONS = [:vms, :hosts, :ems_clusters, :ext_management_systems, :storages, :container_nodes, :container_groups,
```

Comment:
@zeari do you need to keep `:container_groups` too? Or we can drop it (also method below). For example I see that in `rollup.rb` you're replacing what we had (instead of adding).